### PR TITLE
Add package xbrightness

### PR DESCRIPTION
**Pull request by k0001**
`xbrightness` allows controlling by software the screen gamma and brigthness settings for individual color channels.

Personally, I use this package as a free replacement for the pleasant colors achieved by [f.lux](https://justgetflux.com), by executing `xbrightness 65535 49601 33924`.

### CODE REVIEW

**Comment by lethalman** [comment]
Don't use a custom builder.sh please. The buildPhase is already `make $makeFlags` you don't need to reimplement it. You can specify configurePhase and installPhase directly in the derivation.

**Comment by ocharles** [comment]
We should try and be a little more helpful when people are sending their first pull request, especially as we have good documentation on this :) 

We have a phase of `stdenv.mkDerivation` called [`configurePhase`](http://nixos.org/releases/nixpkgs/nixpkgs-15.05pre55774.b6f8d1f/manual/#idm140351253493472). As mentioned in the fourth paragraph [here](http://nixos.org/releases/nixpkgs/nixpkgs-15.05pre55774.b6f8d1f/manual/#idm140351253930688).

**Comment by peti** [comment]
Very useful tool! :+1:

**Comment by ocharles** [comment]
@k0001 Btw, you might be interested in `services.redshift`, which is another replacement for F.lux.

**Comment by k0001** [comment]
Thank you all for the criticism and guidance, I just implemented your suggestions.

File: pkgs/tools/X11/xbrightness/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, x11, xlibs }:
```
Comment (lethalman) [inline pkgs/tools/X11/xbrightness/default.nix]: x11 is unused

Comment (k0001) [inline pkgs/tools/X11/xbrightness/default.nix]: Thank you. I just fixed it.

**Comment by lethalman** [comment]
Travis passed, thanks and welcome :)

**Pull request merged**